spec/indico_batch_spec.rb

```diff
@@ -21,7 +21,9 @@
   it 'should access a private cloud' do
     expected_keys = Set.new(%w(Conservative Green Liberal Libertarian))
     data = ['Guns don\'t kill people.', ' People kill people.']
+    Indico.cloud_protocol = 'http://'
```

Comment:
I've got a bit more downtime while things are uploading, so I'll snag this edit!